web/api/README.md

```diff
@@ -0,0 +1,44 @@
+# CodeChecker server Thrift API
+
+This directory contains the API IDL files and the generated API stubs for CodeChecker.
+[Apache Thrift](https://thrift.apache.org/) is used to generate the stubs
+for various programming languages.
+
+The Thrift compiler is executed inside a [Docker](https://www.docker.com/)
+container so docker needs to be installed to generate the stubs.
+
+## API change workflow:
+
+Assuming the current api version is 6.24.0 and no breaking change was
```

Comment:
Please highlight the keywords, important things etc. with back tick in the documentation, so it will be highlighted and easier to read:
```suggestion
Assuming the current api version is `6.24.0` and no breaking change was
```